src/index.js

```diff
@@ -48,11 +47,12 @@ exports.resolve = function resolver(
   config?: ResolverConfig
 ): ResolverResult {
   const jestConfig = getJestConfig(config, file);
-  if (!isTestFile(jestConfig, file)) {
-    return NOTFOUND;
-  }
   const pathToResolve = applyModuleNameMapper(jestConfig, source) || source;
-  const resolvedPath = resolvePath(jestConfig, path.dirname(file), pathToResolve);
+  const resolvedPath = resolvePath(
```

Comment:
This change (as well as one below) was to satisfy an eslint error I was seeing in my editor thanks to the "prettier/prettier" rule config in this project.